### DISCUSSION TITLE
New update notification does not work?

### DISCUSSION THREAD

**Discussion by zivkovic** [immich-app/immich#6532] [Q&A] [answered]
It used to work, but now for some time I do not get a notification after opening immich web. I have the version check enabled, but it still does not work... Does anyone have any ideas why this would be?

Current immich version: v1.91.4

![image](https://github.com/immich-app/immich/assets/2651524/62acbe7a-62bd-4c6f-b6e8-6aa58c07724f)

**Comment by schuhbacca** [comment]
The version check is periodic. Every hour or something like that. Did you wait long enough?

**Reply by zivkovic** [reply]
I have the server running  24/7 and the current version is running for more than a week already. Or did you mean if I left the webpage open this long?

**Reply by schuhbacca** [reply]
No, I mean the server that is running 24/7 will check about every hour I believe for new version. So if a new version is pushed it could take up to a little over an hour to get a notification in the web client or mobile app.

**Reply by zivkovic** [reply]
I thought so too, but still nothing.

**Reply by zivkovic** [reply, marked as answer]
Just restarted the service and now the notification worked... No idea why it bugged out. Thanks for the help.
